br/pkg/lightning/restore/tidb.go

```diff
@@ -272,10 +272,10 @@ func LoadSchemaInfo(
 			tableInfo := &checkpoints.TidbTableInfo{
 				ID:   tblInfo.ID,
 				DB:   schema.Name,
-				Name: tableName,
+				Name: tbl.Name,
```

Comment:
Please add a log here to describe why we need to use `tbl.Name` instead of `tableName`